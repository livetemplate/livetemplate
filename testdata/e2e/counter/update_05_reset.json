{
  "2": "0",
  "3": "zero",
  "4": "<p>Counter is zero</p>",
  "5": "2023-01-01 10:25:00"
}
