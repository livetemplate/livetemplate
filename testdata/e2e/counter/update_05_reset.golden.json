{
  "1": "0",
  "2": "zero",
  "3": "\u003cp\u003eCounter is zero\u003c/p\u003e",
  "4": "2023-01-01 10:25:00"
}